base_app/main.yml

```diff
@@ -0,0 +1,18 @@
+name: CI Angular app through Github Actions
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      ‐ uses: actions/checkout@v2
+      ‐ name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      ‐ name: Setup
+        run: npm ci
+
+      ‐ name: Test
+        run: |
+          npm test -- --no-watch --no-progress --browsers=ChromeHeadlessCI
```

Comment:
What's going on with this file? Looks like an attempt to run things in CI?

Reply:
Removed - haven't tested CI yet.  Will add back later after it's been tested.